dpnp/backend/kernels/dpnp_krnl_sorting.cpp

```diff
@@ -172,7 +172,7 @@ void dpnp_partition_c(void* array1_in, void* array2_in, void* result1, const siz
 
     event.wait();
 
-    dpnp_memory_free_c(shape);
+        dpnp_memory_free_c(shape);
```

Comment:
@shssf 
it seems that stylechecker has problems

Reply:
it depends on tools version because diffrent versions has diffrent default parameters. Previously, I tried to fix stylechecker parameters in configuration file but number of differences are grow with time. Looks like we have some new parameter which needs to be fixed.